scripts/install/setup.sh

```diff
@@ -341,6 +341,7 @@ if [ -z "$EXISTING_CLOUD_FUNCTION" ]; then
   cat $PARENT_DIR/spinnaker-for-gcp/scripts/install/spinnakerAuditLog/index_js.template | envsubst > $PARENT_DIR/spinnaker-for-gcp/scripts/install/spinnakerAuditLog/index.js
   gcloud functions deploy $CLOUD_FUNCTION_NAME --source $PARENT_DIR/spinnaker-for-gcp/scripts/install/spinnakerAuditLog \
     --trigger-http --memory 2048MB --runtime nodejs8 --allow-unauthenticated --project $PROJECT_ID --region $REGION
+  gcloud alpha functions add-iam-policy-binding $CLOUD_FUNCTION_NAME --region=$REGION --member=allUsers --role=roles/cloudfunctions.invoker
```

Comment:
I believe this also needs the `--project` argument.

Reply:
And could probably use a space as the key/value separator instead of `=`, to fit in with the neighborhood.